configure.ac

```diff
@@ -115,7 +115,7 @@ AC_SUBST([HARDEN_CFLAGS])
 AC_SUBST([HARDEN_LDFLAGS])
 
 # Also check for a few non-hardening-related flags.
-MISC_CXXFLAGS=""
+MISC_CXXFLAGS="-std=c++11"
```

Comment:
I don’t believe the project has made a decision about whether we are ready to move to C++11; support is still lacking on some of the older systems where Mosh may be used. But if we do, it should probably be turned on via the [Autoconf macro `AX_CXX_COMPILE_STDCXX_11`](https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx_11.html), which supports multiple compilers.